GUIDE.md

```diff
@@ -1,50 +1,50 @@
+Interactor receives a request for an operation on data from the ViewController with the Request and submits the data to the Presenter using Response.
 
 ## Worker
-Реализует переиспользуемую часть бизнес логики (например: алгоритм фильтрация данных).
+Implements the reused part of the business logic (for example: algorithm for data filtering).
 
 ## Presenter
-Содержит логику подготовки данных для отображения (например: форматирование данных (номера телефона, суммы), локализация и т.д.) и передает их во ViewController посредством ViewModel.
+Contains the logic for preparing data for display (for example: data formatting (phone number, sum), localization, etc.) and passes them to ViewController via ViewModel.
 
 ## ViewController
-Отвечает за отображение состояния, полученого от Presenter и взаимодействует с пользователем. ViewController содержит в себе только одну view, которая создается в методе loadView(). Он не управляет расположением view. Вся необходимая верстка уже должна быть в отдельно реализованной view. Однако ViewController выступает делегатом для всего view слоя. Так он знает, когда изменился текст в TextField или пользователь нажал на ячейку.
+Responsible for displaying the status received from Presenter and interacting with the user. ViewController contains only one view, which is created in the loadView () method. It does not control the location of the view. All necessary layout should be already presented in a separately implemented view. However, ViewController acts as a delegate for the entire view layer. So it knows when the Text in TextField was changed or the user clicked on the cell.
 
 ```
-override func loadView() {
-    view = CatalogListView(delegate: self)
+override func loadView () {
+    view = CatalogListView (delegate: self)
 }
 ```
 
 ## Provider
-Абстракция отвечающая за доступ к данным. Инкапсулирует в себе обращение к Service для получения данных из сети (или других источников) и/или DataStore, если данные должны кэшироваться.
+Abstraction responsible for accessing data. Encapsulates a call to Service to retrieve data from the network (or other sources) and/or DataStore if the data should be cached.
 
 ## DataStore
-Объект абстрагирующий долгосрочное хранение данных. 
+Object abstracting long-term data storage.
 
 ```
 class CatalogDataStore {
-    static let shared = CatalogDataStore()
+    static let shared = CatalogDataStore ()
 
-    var catalogModels: [CatalogObjectType: [CatalogModel]] = [:]
+    var catalogModels: [CatalogObjectType: [CatalogModel]] = [:]
 }
 ```
 
 ## ViewModel
-Содержит в себе полностью подготовленные для отображения данные для View (каждой отдельной subview). ViewModel — это DTO и не содержит в себе никакой логики.
+Contains completely prepared for display data for View (each separate subview). ViewModel is a DTO and does not contain any logic.
```

Comment:
ViewModel contains completely prepared data for display. ViewModel is a DTO and does not contain any logic.